xarray/tests/test_backends.py

```diff
@@ -4382,6 +4382,23 @@ def roundtrip_dir(
         ) as ds:
             yield ds
 
+    @requires_dask
+    def test_default_zarr_fill_value(self):
+        inputs = xr.Dataset({"floats": ("x", [1.0]), "ints": ("x", [1])}).chunk()
+        expected = xr.Dataset({"floats": ("x", [np.nan]), "ints": ("x", [0])})
+        with self.temp_dir() as (d, store):
+            inputs.to_zarr(store, compute=False)
+            with open_dataset(store) as on_disk:
+                assert np.isnan(on_disk.variables["floats"].encoding["_FillValue"])
```

Comment:
Not a blocker, but it seems a bit unnecessary to have `_FillValue` set at all in this scenario, doesn't it? We can just write NaNs directly?

Reply:
Agreed. Created https://github.com/pydata/xarray/issues/10781 so we don't forget about this.